tariff-management-service/app/src/main/java/com/microservices/tariffmanagement/entity/TariffEntity.java

```diff
@@ -25,9 +25,9 @@
     //in millis
     private long washingTime;
 
-    public TariffEntity(TariffDto tariffDto) {
-        this.name = tariffDto.getName();
-        this.price = tariffDto.getPrice();
-        this.washingTime = tariffDto.getWashingTime();
+    public TariffEntity(CreationTariffDto creationTariffDto) {
+        this.name = creationTariffDto.getName();
```

Comment:
please, check if creationDto is not null

Reply:
Here or in the service? If here, what should I do in case of null? Throw a NullPointer?

Reply:
Add null checks in constructors like `public TariffEntity(TariffDto tariffDto)` and same for `TariffDto (TariffEntity tariffEntity)`

Reply:
Objects.requireNotNull

Reply:
Hope, NonNull will be ok too

Reply:
The only concern is that there are so many different NonNull annotations is java world which are acting in different ways... Ok, let it be there